model/src/main/kotlin/OrtResult.kt

```diff
@@ -400,6 +400,23 @@ data class OrtResult(
     fun replaceConfig(config: RepositoryConfiguration): OrtResult =
         copy(repository = repository.copy(config = config)).also { it.data += data }
 
+    /**
+     * Return a copy of this [OrtResult] with the [PackageCurations]s replaced by the given [curations].
```

Comment:
So, after reading the implementation it's now clear that *all* curations are removed and then *only* the passed curations are applied. I'm just wondering whether this is really what we want. Wouldn't we rather want to *amend* existing curations with the passed ones, as a test for adding new curations to the list?

Reply:
> So, after reading the implementation it's now clear that _all_ curations are removed and then _only_ the passed curations are applied. I'm just wondering whether this is really what we want. 

I am using this for re-applying the global curations file and I think this scenario is a good reason to want it implemented like it is now.

> Wouldn't we rather want to _amend_ existing curations with the passed ones, as a test for adding new curations to the list?

I don't see any need for that - do You? ...the implementation I anticipate to be more complicated.

Reply:
> I don't see any need for that - do You?

No, I just wanted to get your input on whether you consider that to be a valid support use-case.